arbor/include/arbor/util/optional.hpp

```diff
@@ -165,6 +166,14 @@ namespace detail {
             }
             set = false;
         }
+
+        template <typename Y>
+        void emplace(Y&& y) {
+            if (set)
+                data.destruct();
```

Comment:
Just in terms of the Arbor code style, we should have some braces around the body of the `if`.

Reply:
Thanks. Just pushed the adjustments.